test/k8sT/Services.go

```diff
@@ -1330,7 +1330,16 @@ var _ = Describe("K8sServicesTest", func() {
 
 		// Net-next and not old versions, because of LRU requirement.
 		SkipItIf(helpers.DoesNotRunOnNetNextOr419Kernel, "Supports IPv4 fragments", func() {
-			DeployCiliumAndDNS(kubectl, ciliumFilename)
+			var options map[string]string
+			if helpers.RunsOn419Kernel() {
+				// On 4.19, because we're running with
+				// kube-proxy-replacement=strict, we need to choose between
+				// debug mode and fragment tracking support to avoid complexity
+				// issues.
+				options["global.fragmentTracking"] = "true"
+				options["global.debug.enabled"] = "false"
+			}
+			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, options)
```

Comment:
Nit: Even if it defaults to `true`, we could maybe set `options["global.fragmentTracking"]` to `true` explicitly for all kernels for this test? (I should have done that and the Helm option, my bad.)
```suggestion
			var options map[string]string
			// On 4.19, because we're running with
			// kube-proxy-replacement=strict, we need to choose between
			// debug mode and fragment tracking support to avoid complexity
			// issues.
			options["global.fragmentTracking"] = "true"
			options["global.debug.enabled"] = !helpers.RunsOn419Kernel()
			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, options)
```